src/main/java/org/betonquest/betonquest/BetonQuest.java

```diff
@@ -1548,6 +1548,10 @@ public String getVariableValue(final String packName, final String name, final P
                 log.warn(pack, "Could not resolve variable '" + name + "'.");
                 return "";
             }
+            if (profile == null && !var.isStaticness()) {
```

Comment:
I think there is something disappeared?
```suggestion
            if ((playerID == null || playerID.isBlank()) && !var.isStaticness()) {
```

Reply:
yeah i figured that it is sufficient to just check whether the profile is `null`? or is it possible to have a profile without a valid player uuid?

Reply:
note, a profile always has a valid uuid